fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -105,9 +107,9 @@ const createElection = (
   // compute the id for all questions and add the write_in property
   const questionsWithId = questions.map((item) =>
     Question.fromState({
-      id: Hash.fromArray(EventTags.QUESTION, electionId, item.question).toString(),
-      question: item.question,
-      ballot_options: item.ballot_options,
+      id: Hash.fromArray(EventTags.QUESTION, electionId, item.question.trim()).toString(),
+      question: item.question.trim(),
+      ballot_options: item.ballot_options.map((s) => s.trim()),
```

Comment:
Same comment as above. Try to create the "ready for processing" values using `useMemo`, perform the checks on these and then directly use them here.

If you perform processing here you cannot be sure whether the resulting data satisfies the desired properties.